python/cac_tripplanner/templates/admin/cac-geocoding-map.js

```diff
@@ -0,0 +1,22 @@
+{% extends 'gis/admin/osm.js' %}
+
+{% block extra_layers %}
+
+var $address = $('[name=address]');
+var module = {{ module }};
+var map = module.map;
+var projLatLng = new OpenLayers.Projection('EPSG:4326');
+var projWM = new OpenLayers.Projection('EPSG:900913'); // 3857 doesn't work here for some reason
```

Comment:
Looks like it may be necessary to [declare an alias](http://docs.openlayers.org/library/spherical_mercator.html#sphericalmercator-and-epsg-aliases) for 3857 in OpenLayers.  (I'm fine with the Google code here.)